packages/dapp-toolkit/src/connect-button/connect-button-client.ts

```diff
@@ -95,6 +95,17 @@ export const ConnectButtonClient = (input: {
 
           connectButtonElement.enableMobile = enableMobile
 
+          if (transport?.sessionChange$)
```

Comment:
Maybe let's follow the convention which we already have?

1. Create subject for each input
2. Move subscription out of initialization function

Reply:
Added a subject for linking. Didn't see a point to include pristine and showPopoverMenu as they are not the desired state, just additional attributes that needed for linking.